backend/src/core/loaders/express.loader.ts

```diff
@@ -29,9 +32,23 @@ morgan.token('user-id', userId)
 // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
 // @ts-ignore
 const loggerMiddleware = morgan(config.get('MORGAN_LOG_FORMAT'), {
-  stream: logger.stream,
+  stream: loggerInstamce.stream,
```

Comment:
spelling... but actually for our sanity, could we add a getter and change this to 
`Logger.getStream()`